src/main/java/it/gov/pagopa/cgn/portal/repository/DiscountBucketCodeSummaryRepository.java

```diff
@@ -12,4 +12,6 @@
     DiscountBucketCodeSummaryEntity findByDiscount(DiscountEntity discount);
 
     List<DiscountBucketCodeSummaryEntity> findAllByExpiredAtIsNull();
+
+    List<DiscountBucketCodeSummaryEntity> findAllByExpiredAtIsNotNull();
```

Comment:
Why not expiredAt greater than a given Date (namely the date calculated from `suspendDiscountsWithoutAvailableBucketCodesAfterDays`)

Reply:
@AleDore I should also get only those one with a published discount to avoid getting them all over and over

Reply:
fixed in https://github.com/pagopa/cgn-onboarding-portal-backend/pull/111/commits/ee55476038fb0de8ebce2c18b900231a810ce8b8